linux_kinect/Bit[] data
string args
---
geometry_msgs/Point[] points
linux_kinect/Cognition[] cognitions
linux_kinect/Bit[] bits
bool status